.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/TimePeriodParserChs.cs

```diff
@@ -112,44 +112,40 @@ private bool GetMatchedTimexRange(string text, out string timex, out int beginHo
             beginHour = 0;
             endHour = 0;
             endMin = 0;
-            
-            if (trimmedText.EndsWith(Constants.CN_MORNING))
+
+            var timeOfDay = "";
+            if (trimmedText.EndsWith(DateTimeDefinitions.Morning))

```

Comment:
In this code to determine tod, timeOfDay should receive values representing the tod concept (maybe even directly like TMO), not a string in a given language.